### ISSUE TITLE
[Feature request] Disable save notification icon 

### ISSUE THREAD

**Issue by ks202005** [andrei-drexler/ironwail#198]
In Ironwail, there seems to be an added save icon that appears in the top-right and displays a pentagram (I believe this icon was used in DOS and WinQuake to indicate loading and saving as well?) with the name of the save.

It'd be nice to disable this, since I'm used to it not appearing in QSS and vkQuake.

**Comment by SkullGunner** [comment]
Agreed, would also like the autosave text being hidden by default, too many notifications for my liking!
